src/templates/base/2020/index.html

```diff
@@ -22,6 +22,40 @@
 {% block styles %}
   {{ super() }}
   <link rel="stylesheet" href="/static/css/index.css?v=20200630133215">
+  <style>
+.intro {
+  max-width: 600px;
+}
+.intro p {
+  font-style: italic;
+}
+.intro p:first-child {
+  margin-bottom: 20px;
+}
+.methodology-container {
+  margin-top: 150px;
+}
+.methodology-characters {
+  z-index: unset;
+}
+@media (max-width: 600px) {
+  .intro h2 {
+    font-size: 2rem;
+  }
+  .intro h2 b {
+    display: block;
+  }
+  .intro-image-wrapper {
+    margin-right: -100px;
+  }
+  .methodology-container {
+    margin-top: 200px;
+    padding: 0;
+  }
+}
+
```

Comment:
Why do we need all this? And why italics?

Seems to work with just below and is more consistent with 2019:
```suggestion
.methodology-characters {
  z-index: unset;
}
```

Reply:
These are only needed for the temporary landing page and can be removed when we launch the full contents. With most of the content missing, the background image gets cut off abruptly, so I added the methodology characters and adjusted the styles to make the minimal content fit together better.

Reply:
Yup all of these styles were added intentionally and customized for the temp landing page. Many were small tweaks to the sizing of contents/containers so that the text and images are all better aligned. Let me know if there are any styles you feel are unnecessary.

Reply:
Not too worried as only temporary so happy to leave. Though why italics? Think that’s really needed to differentiate it as a holding page?